package.json

```diff
@@ -25,9 +25,8 @@
     "await-semaphore": "^0.1.3",
     "commander": "^2.14.1",
     "debug": "^3.1.0",
-    "noble": "^1.9.0",
     "pc-nrfjprog-js": "^1.2.0",
-    "serialport": "^6.0.5",
+    "serialport": "^6.1.1",
```

Comment:
Please check if this serialport dependency does work on Windows as well, I don't think this one includes the serialnumber fix.